tests/controllers/test_uaa_controller.py

```diff
@@ -24,13 +24,13 @@
 with open(f"{project_root}/test_data/uaa/email_search_user.json") as fp:
     uaa_user_search_email = json.load(fp)
 
-user_id = "fe2dc842-b3b3-4647-8317-858dab82ab94"
+user_id = "some.one@ons.gov.uk"
```

Comment:
Does the user_id really change with this?  Each user should still have a uuid that identifies them that's separate from the username and email

Reply:
Oh yeah, I have mistaken user_id and user_name on a few, darn